contracts/libraries/FixedPoint.sol

```diff
@@ -19,8 +19,9 @@ library FixedPoint {
     }
 
     uint8 private constant RESOLUTION = 112;
-    uint private constant Q112 = uint(1) << RESOLUTION;
-    uint private constant Q224 = Q112 << RESOLUTION;
+    uint  private constant Q112 = uint(1) << RESOLUTION;
+    uint  private constant Q224 = Q112 << RESOLUTION;
+    uint  private constant LOWER_MASK = 0xffffffffffffffffffffffffffff; // decimal of UQ*x112 (lower 112 bits)
```

Comment:
i originally had this as a `uint224` to pack with the `uint8`, but because it's a constant i believe it just gets inserted directly into the bytecode, hence `uint`